features/step_definitions/user_steps.rb

```diff
@@ -402,3 +406,8 @@
   link = user_path(user)
   page.find(:css, %Q{a[href="#{link}?tab=activity"]}).trigger('click')
 end
+
+Then(/^I should not exist as a user$/) do
+  user = User.find_by(first_name: 'Anders')
+  expect(user).to be_nil
```

Comment:
let's put that into a refactoring ticket https://github.com/AgileVentures/WebsiteOne/issues/1455